release/release.go

```diff
@@ -55,8 +55,24 @@ var (
 // ForceDevMode returns true if the distribution doesn't implement required
 // security features for confinement and devmode is forced.
 func (o *OS) ForceDevMode() bool {
-	level, _ := apparmor.ProbeKernel().Evaluate()
-	return level != apparmor.Full
+	return AppArmorSupportLevel() != apparmor.FullSupport
```

Comment:
I realize that you are returning the equivalent of before, but this (forcing devmode if not full support) seems inconsistent with interfaces/backends/backends.go init() where now add the apparmor backend if there is any level of support. Perhaps at least the comment needs to be adjusted?

Reply:
Let's please not change this here. ForceDevMode as an idea should hopefully die altogether, but this PR is lingering for too long on issues which are related to its implementation. Let's keep unrelated fixes for follow ups please.

Reply:
I agree it should be in a future PR, but felt the inconsistency should at least be pointed out (so we know to change it going forward! :) and that perhaps the comment should change.